perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -637,7 +637,36 @@ List of names of pages to display on the admin interface
 
 =cut
 
-sub admin_pages { 0 }
+sub admin_pages {
```

Comment:
If we're having this in the Default, do we get rid of the 'default' in set_allowed_pages?